docs/source/architecture/rest_api.rst

```diff
@@ -0,0 +1,378 @@
+********
+REST API
+********
+
+*Hyperledger Sawtooth* provides a pragmatic RESTish API for clients to interact
+with a validator using common HTTP/JSON standards. An entirely separate process,
```

Comment:
Suggest changing the sentence starting with "An entirely separate process..." to "It executed in an entirely separate process, which, once running, allows transaction to be submitted...etc"